Bendomey/project-mfoni
# feat: Add/Remove Contents in a Collection

## PR Name or Description

feat: Add/Remove Contents in a Collection

## Overview

Add a functionality where creators can add/remove contents in a collection

## Detailed Technical Change

- Added dialogs for adding/removing contents
- Creators can add up to 50 contents per collections.

## Testing Approach & Result

- Login
- Upload a content
- Create a collection
- View the collection and then you should see a button "Add Contents"

## Related Work

N/A

## Documentation

N/A
